solr/core/src/test/org/apache/solr/cli/ZkSubcommandsTest.java

```diff
@@ -507,10 +507,8 @@ public void testGetFile() throws Exception {
     Path file =
         tmpDir.resolve("solrtest-getfile-" + this.getClass().getName() + "-" + System.nanoTime());
 
-    String[] args =
-        new String[] {
-          "cp", "-z", zkServer.getZkAddress(), "zk:" + getNode, file.toAbsolutePath().toString()
-        };
+    // Not setting --zk-host, will fall back to sysProp 'zkHost'
+    String[] args = new String[] {"cp", "zk:" + getNode, file.toAbsolutePath().toString()};
```

Comment:
:-(.  See me rant email ;-)

Reply:
Did not find your mail. I like env vars also in cli tools :)